<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Servo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="ConnectButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_Connect">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="StatusLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnected!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_Serv_1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Servo Rotation:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame_Serv_1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="label_Serv_F_Val_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_Serv_F_Deg_1">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Degree</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDial" name="dial_Serv_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>321</width>
      <height>321</height>
     </rect>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
   </widget>
  </widget>
  <action name="actionConnectButtonAction">
   <property name="text">
    <string>ConnectButtonAction</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>ConnectButton</tabstop>
  <tabstop>dial_Serv_1</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>dial_Serv_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_Serv_F_Val_1</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ConnectButton</sender>
   <signal>clicked()</signal>
   <receiver>actionConnectButtonAction</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
